internal/func.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 const fl = require('../');
-const equality = (x, y) => typeof x[fl.equals] === 'function' ? x[fl.equals](y) : x === y;
+const equality = (x, y) => x && typeof x[fl.equals] === 'function' ? x[fl.equals](y) : x === y;
```

Comment:
This new function will not strictly return a Boolean: `equality(null) === null`. Not sure if that's a problem, but if it is, I would suggest:

```js
const equality = (x, y) => Boolean(x) && ...
```